thirdparty/build-thirdparty.sh

```diff
@@ -1180,6 +1180,16 @@ build_clucene() {
     fi
 }
 
+# avx2ki
+build_avx2ki() {
+    check_if_source_exist $AVX2KI_SOURCE
+    cd $TP_SOURCE_DIR/$AVX2KI_SOURCE
+    rpm -ivh boostkit-ksl-2.1.0-1.aarch64.rpm
```

Comment:
shall use `cpio` to extract files from the rpm and install them into THIRDPARTY/installed location.